src/plugins/orangehrmCorePlugin/Service/ConfigService.php

```diff
@@ -501,14 +501,18 @@ public function getTimesheetTimeFormatConfig(): ?string
      */
     public function getLDAPSetting(): LDAPSetting
     {
-        return LDAPSetting::fromString($this->_getConfigValue(self::KEY_LDAP_SETTINGS));
+        $ldapSetting = $this->_getConfigValue(self::KEY_LDAP_SETTINGS);
+        if (is_null($ldapSetting)) {
+            return new LDAPSetting('localhost', 389, 'OpenLDAP', 'none', 'dc=example,dc=com');
```

Comment:
```suggestion
            return new LDAPSetting('localhost', 389, 'OpenLDAP', 'none', '');
```
shall we set this empty :thinking: